src/constants/MarkdownFiles/posts/2025-10-04-adding-dark-mode-to-sugar-labs.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Adding Dark Mode to a Sugar Labs Website"
+excerpt: "I added dark mode to a Sugar Labs web project—here’s the problem I hit, how I debugged it, and what I learned."
+category: "DEVELOPER NEWS"
+date: "2025-10-04"
+slug: "adding-dark-mode-to-sugar-labs"
+author: "@/constants/MarkdownFiles/authors/syed-khubayb-ur-rahman.md"
+tags: "dark-mode,accessibility,frontend,Sugar Labs"
+image: "assets/Images/dark-mode.webp"
+---
+
+<!-- markdownlint-disable -->
+
+# Adding Dark Mode to a Sugar Labs Website
+
+## Introduction
+Sugar Labs builds open-source tools that make learning playful and accessible. I recently contributed dark mode to a Sugar Labs web project. This post outlines the problem I encountered, how I debugged and fixed it, and the key lessons I learned so others can ship accessible theming with confidence.
+
+## The Problem
+The Sugar Labs website originally lacked a Dark Mode option, which made browsing less comfortable in low-light conditions. This was particularly noticeable for developers and users who preferred darker themes.
+
+Without dark mode, the bright interface could cause eye strain and reduce accessibility for users with light sensitivity.
```

Comment:
Please be consistent with capitalization. "Dark Mode" or "dark mode". Don't switch between the two.